bootstrap.php

```diff
@@ -1,11 +1,17 @@
 <?php
 
+use Dotenv\Dotenv;
 use staabm\PHPStanDba\QueryReflection\QueryReflection;
 use staabm\PHPStanDba\QueryReflection\RuntimeConfiguration;
 use staabm\PHPStanDba\Tests\ReflectorFactory;
 
 require_once __DIR__.'/vendor/autoload.php';
 
+if (false === getenv('GITHUB_ACTION')) {
```

Comment:
just wanted to add to the rest of bootstraps.. the if might be extra

Reply:
did you mean there is something missing?

Reply:
it's fine))